.gitignore

```diff
@@ -1,4 +1,6 @@
+dist/
 node_modules/
 pkg/
 result
 target/
+*~
```

Comment:
https://gist.github.com/subfuzion/db7f57fff2fb6998a16c#file-global-gitignore-md

Reply:
Thanks, I didn't know about global ignoring.  I'll start using it